src/backend/distributed/deparser/qualify_sequence_stmt.c

```diff
@@ -59,7 +62,8 @@ QualifyAlterSequenceSchemaStmt(Node *node)
 
 	if (seq->schemaname == NULL)
 	{
-		Oid schemaOid = RangeVarGetCreationNamespace(seq);
+		Oid seqOid = RangeVarGetRelid(seq, NoLock, stmt->missing_ok);
+		Oid schemaOid = get_rel_namespace(seqOid);
```

Comment:
given we pass `missing_ok` means that there is a case where `seqOid` can be `InvalidOid`. once we have that, we should _not_ lookup the schema for the sequence as we know it doesn't exist.